src/main/java/core/basesyntax/service/AmountFromFile.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface AmountFromFile {
+    List<Integer> getAmountInShop(List<String> fruits, List<FruitTransaction> fruitTransactions);
```

Comment:
The parameter 'fruitTransactions' uses a class 'FruitTransaction' that hasn't been imported. Ensure that you import the necessary classes.